example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -44,6 +44,11 @@ leaderElection:
   renewDeadline: 10s
   retryPeriod: 2s
   resourceLock: configmaps
+# `discovery` allows specifying options for the cached discovery client.
+# discovery:
```

Comment:
Can you please implement the possibility to configure that in the Gardener Helm chart?